WcaOnRails/config/routes.rb

```diff
@@ -138,6 +140,7 @@
 
   get 'competitions/:id/payment_setup' => 'competitions#payment_setup', as: :competitions_payment_setup
   get 'stripe-connect' => 'competitions#stripe_connect', as: :competitions_stripe_connect
+  get 'competitions/:id/paypal-return' => 'competitions#paypal_return', as: :competitions_paypal_return
```

Comment:
Why was this return URL route not included in the previous PR?
Is this a merge whoopsie or a general whoopsie?

Reply:
It was included in the previous PR under the `unless Rails.env.production?` clause - the actual whoopsie was that all the routes were duplicated